src/main/java/com/company/KafkaProducerTest.java

```diff
@@ -45,11 +45,15 @@ public static void main(String[] args) {
 //    for (int i = 0; i < 100; i++) {
```

Comment:
This code isn't used and if we want to see it we can go back in in history in source control. As such let's delete it.